src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -1926,13 +1916,6 @@ emit_sri_vector (MonoCompile *cfg, MonoMethod *cmethod, MonoMethodSignature *fsi
 		if (!is_element_type_primitive (fsig->params [0]))
 			return NULL;
 		int op = id == SN_GetLower ? OP_XLOWER : OP_XUPPER;
-
-#ifdef TARGET_AMD64
-		if (!COMPILE_LLVM (cfg))
```

Comment:
This is an AMD64 block.

Reply:
Good catch. It's easy to confuse `ARM64` and `AMD64` in code.